docs/playbook/Module-0.md

```diff
@@ -0,0 +1,132 @@
+- Load 1 or more Dataset from S3 into a coding environment, local and on AWS Glue
+- Learn how to test your Data Transformation in the Local Notebooking Environment ( saves money and is much faster to debug )
+- Learn how to do transformations in Spark syntax (including Joins)
+- Add a Glue job within the AWS console
+- Save the dataset after transformation into the refined zone
+
+### Module 3
+This module is intended for people who want to schedule or automate data transformations
+After completing this module, you should be able to do the following:
+- Deploy a Glue Job into the Data Platform Production environment, with scheduler to automate the process
+- Deploy a Glue job using Terraform
+</details>
+
+## Why we are moving away from old ways of work, and working using a Data Platform
+Please visit and read : https://playbook.hackney.gov.uk/Data-Platform-Playbook/
+
+# What are the prerequisites for these training modules  
+<details>
+    <summary>Click to Expand</summary>
+
+##Introduction: Experience, skills, software and equipment
+
+To get the most out of these modules, you must be comfortable with the idea of coding. It is fine to be a beginner in Python and Terraform, but you should at least have a bit of experience with a programming language (such as SQL or formulas in Excel/Sheets). 
+
+You’ll also need accounts and permissions to access some platforms. In Module 2 you will use a local environment, so you’ll also need a Windows machine or a Mac and permissions to install software on it (though we have plans to develop the platform so that this is no longer the case).
+
+You may have to contact IT to install some prerequisite software if you do not have the permissions to do it yourself
+
+## Pre-requisites Table
+
+Here are the details on these requirements by module.
+
+| Module | Github | Notebook Environment | AWS Access |
+|---|---|---|---|
+| Module 1 | X | | |
+| Module 2 | X | X| X|
+| Module 3 | X | | X|
+
+</details>
+
+# Tools that are used with the Data Platform
+
+<details>
+    <summary>Click to Expand</summary>
+
+## GitHub: Online repository for code where all Hackney codebase is stored
+
+Github is where the Data Platform repository is stored. In particular, it is where the Terraform scripts are, which is the code which controls and manages the Data Platform resources in AWS. We also store the code for the Playbook there
+You can refer to this playbook entry for more information : https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/getting-set-up/using-github
+
+### Benefits of using these tools 
+Using Github means we have version control on our code, we also have a way to verify code and making sure code is suitable before it is Live
+
+
+## The notebook environment
+
+Essentially a Jupyter server so that we can run Jupyter Notebooks. Jupyter notebooks are Python scripts which run in cells, which makes it easier to develop and try things out
```

Comment:
```suggestion
Essentially a Jupyter server so that we can run Jupyter Notebooks. Jupyter notebooks are Python scripts that run in cells, which makes it easier to develop and try things out.
```